crates/ironrdp-client/Cargo.toml

```diff
@@ -56,8 +56,10 @@ anyhow = "1"
 smallvec = "1.11"
 tap = "1"
 semver = "1"
+reqwest = "0.11.22"
+url = "2.4.1"
 
 [target.'cfg(windows)'.dependencies]
 windows = { version = "0.48", features = [
-    "Win32_Foundation",
+    "Win32_Foundation"
```

Comment:
FYI, trailing commas are the convention in Rust (it makes it easier to insert new values later without modifying both lines)